.github/workflows/notebook-test.yml

```diff
@@ -46,35 +42,11 @@ jobs:
             docs/transpile/representing_quantum_computers.ipynb
             docs/transpile/transpiler-stages.ipynb
 
-      - name: Install Linux dependencies
-        if: steps.linux-changed-files.outputs.any_changed == 'true'
-        run: |
-          sudo apt-get update
-          sudo apt-get install texlive-pictures texlive-latex-extra poppler-utils graphviz
-
-      - name: Install Python packages
-        # This is to save our account in the next step. Note that the
-        # package will be re-installed during the "Run tox" step.
-        run: pip install qiskit-ibm-runtime tox
-
-      - name: Save IBM Quantum account
-        if: ${{ github.event.pull_request.head.repo.full_name == github.repository }}
-        shell: python
-        run: |
-          # This saves the result for qiskit-ibm-provider too
-          from qiskit_ibm_runtime import QiskitRuntimeService
-          QiskitRuntimeService.save_account(
-              channel="ibm_quantum",
-              instance="ibm-q/open/main",
-              token="${{ secrets.IBM_QUANTUM_TEST_TOKEN }}",
-              set_as_default=True
-          )
-
-      - name: Cache tox environment
-        uses: actions/cache@v3
+      - name: Setup environment
+        uses: ./.github/workflows/reusable/setup-notebook-testing
         with:
-          path: ".tox"
-          key: ${{ hashFiles('scripts/nb-tester/requirements.txt') }}
+          install-linux-deps: ${{ steps.linux-changed-files.outputs.any_changed }}
+          ibm-quantum-token: ${{ secrets.IBM_QUANTUM_TEST_TOKEN }}
```

Comment:
Instead, I think you're supposed to pass it via `secrets`. But maybe not because you're using an action over worfklow?

https://github.com/Qiskit/documentation/blob/9ff00250401453df6a164185e9038f57f6579d42/.github/workflows/preview-docs.yml#L36-37

Reply:
According to [SO](https://stackoverflow.com/a/70111134) (I can't find an official source) actions must use inputs